File: app/src/main/java/com/apextechies/kmaaoapp/login/activity/MainActivity.java
package com.apextechies.kmaaoapp.login.activity;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.RecyclerView;

import com.apextechies.kmaaoapp.R;

import butterknife.BindView;

/**
 * Created by shankar on 25/1/18.
 */

public class MainActivity extends AppCompatActivity {
    @BindView(R.id.rv_applist)
    RecyclerView recyclerView;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.applist_row);
    }
}

File: app/src/main/java/com/apextechies/kmaaoapp/MainActivity.kt
package com.apextechies.kmaaoapp

import android.content.Intent
import android.os.Bundle
import android.support.design.widget.Snackbar
import android.support.v7.app.AppCompatActivity
import android.view.Menu
import android.view.MenuItem
import com.apextechies.kmaaoapp.login.Login
import com.crashlytics.android.Crashlytics
import com.firebase.ui.auth.AuthUI
import com.google.android.gms.tasks.OnCompleteListener
import com.google.android.gms.tasks.Task
import io.fabric.sdk.android.Fabric

import kotlinx.android.synthetic.main.activity_main.*
//import sun.security.krb5.internal.KDCOptions.with



class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Fabric.with(this, Crashlytics())
        setContentView(R.layout.activity_main)
        setSupportActionBar(toolbar)

        fab.setOnClickListener { view ->
            Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
                    .setAction("Action", null).show()
        }
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        // Inflate the menu; this adds items to the action bar if it is present.
        menuInflater.inflate(R.menu.menu_main, menu)
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        // Handle action bar item clicks here. The action bar will
        // automatically handle clicks on the Home/Up button, so long
        // As you specify a parent activity in AndroidManifest.xml.
        return when (item.itemId) {
            R.id.action_settings -> true
            else -> super.onOptionsItemSelected(item)
        }
    }

    override fun onBackPressed() {
        super.onBackPressed()
        AuthUI.getInstance()
                .signOut(this@MainActivity)
                .addOnCompleteListener {
                    startActivity(Intent(this@MainActivity, Login::class.java))
                    finish()
                }
    }
}
